controller/openebs/apiserver.go

```diff
@@ -154,3 +168,23 @@ func (p *Planner) setAPIServerDefaultsIfNotSet() error {
 	}
 	return nil
 }
+
+// updateMayaAPIServerService updates the maya-apiserver-service manifest as per the
+// reconcile.ObservedOpenEBS values.
+func (p *Planner) updateMayaAPIServerService(svc *unstructured.Unstructured) error {
```

Comment:
Why do we need to label the service ? Can't we add one label for every component (deployment, daemon-set, sts) only?

Reply:
This will help in listing all the OpenEBS related services using these labels.

Reply:
Ok